docs/user/managing-devices.md

```diff
@@ -482,6 +487,22 @@ spec:
 [...]
 ```
 
+## Creating Applications
```

Comment:
Like with Building OS Images, this would logically belong in a dedicated section on "Building Application Package Images", as it describes build-time / producer, while this whole section is about run-time / consumer. We can move things around later if necessary, though.

Reply:
Makes sense and these docs will extend in the future.